src/default-xpring-client.ts

```diff
@@ -267,6 +268,17 @@ class DefaultXpringClient implements XpringClientDecorator {
       return false
     }
   }
+
+  // TODO(keefertaylor): Re-enable
+  /* eslint-disable class-methods-use-this */
+  /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
seems like it might be worth adding this lint rule: https://github.com/xpring-eng/xpring-wallet-system/pull/373